request/api.go

```diff
@@ -100,19 +95,36 @@ func (c *APIClient) GetPreSignedUploadURL(ctx context.Context, md5 string) (Sign
 }
 
 func (c *APIClient) DeleteIntegration(ctx context.Context, integrationId string) error {
-	url := fmt.Sprintf("%s/projects/%s/integrations/%s", c.Root, c.ProjectId, integrationId)
+	delURL := fmt.Sprintf("%s/projects/%s/integrations/%s", c.Root, c.ProjectId, integrationId)
 
+	auth, err := c.getAuthHeader(ctx)
+	if err != nil {
+		return err
+	}
+
+	if _, err := c.RequestClient.Delete(ctx, delURL, auth); err != nil { //nolint:bodyclose
+		return fmt.Errorf("error deleting integration: %w", err)
+	}
+
+	logger.Debugf("Deleted integration: %v", integrationId)
+
+	return nil
+}
+
+func (c *APIClient) getAuthHeader(ctx context.Context) (Header, error) {
 	if c.APIKey != nil && *c.APIKey != "" {
 		header := Header{Key: "X-Api-Key", Value: *c.APIKey}
-		if _, err := c.RequestClient.Delete(ctx, url, header); err != nil { //nolint:bodyclose
-			return fmt.Errorf("error deleting integration: %w", err)
-		}
-
-		logger.Debugf("Deleted integration: %v", integrationId)
-	} else {
-		// TODO: Default to token authentication and set Authorization header, instead of failing.
-		logger.Fatal("Must provide an API key in the --key flag")
+
+		return header, nil
 	}
 
-	return nil
+	jwt, err := clerk.FetchJwt(ctx)
+	if err != nil {
+		return Header{}, err
```

Comment:
Should call `logger.Fatalf` with an error instructing the user to either run `amp login` or to provide a key flag to the command.